examples/motoko_examples/phone-book/canisters/azle/index.ts

```diff
@@ -0,0 +1,40 @@
+import { ic, nat64, Opt, Query, Update } from 'azle';
+
+type PerfResult = {
+    wasm_body_only: nat64;
+    wasm_including_prelude: nat64;
+};
+
+let perf_result: Opt<PerfResult> = null;
+
+export function get_perf_result(): Query<Opt<PerfResult>> {
+    return perf_result;
+}
+
+function record_performance(start: nat64, end: nat64): void {
+    perf_result = {
+        wasm_body_only: end - start,
+        wasm_including_prelude: ic.performance_counter(0)
+    };
+}
+
+export type Entry = {
```

Comment:
Is there a reason to export this type?

Reply:
Yes the front end needs it to be exported